### PR TITLE
ARUHA-56: created a draft of nakadi scopes schema;

### PR DESCRIPTION
**Pull request by v-stepanov**
This PR is only for discussion, don't merge please

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/85" target="_blank">ReviewNinja

File: nakadi-scopes.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Resource Type ID: nakadi_event
+Resource Type Name: Nakadi Event
+Owning team: aruha
+Application Scope: nakadi_event.read
```
Comment (rcillo) [inline nakadi-scopes.txt]: I think we need a scope to create/update/list event types as of https://techjira.zalando.net/browse/ARUHA-50 right?

Comment (v-stepanov) [inline nakadi-scopes.txt]: Yeah, true. But that will be another resource. Can be something like `nakadi_event_type` with scopes also `read` and `write`.

Comment (jkliff) [inline nakadi-scopes.txt]: nakadi.event... not nakadi_event.
see https://github.com/zalando/nakadi/pull/86/files#diff-4492c6b26ee0a37518728ec1585420efR112

**Comment by v-stepanov** [comment]
@jkliff @tmuehl @CyberDem0n @rcillo 
Updated the scopes as we agreed

**Pull request closed without merging**